.github/workflows/bsk_pr_checks.yml

```diff
@@ -72,6 +72,7 @@ jobs:
             SharedPackages/BrowserServicesKit/.build/workspace-state.json
           key: ${{ runner.os }}-bsk-spm-${{ env.cache_key_hash }}
           restore-keys: |
+            ${{ runner.os }}-bsk-spm-${{ env.cache_key_hash }}
```

Comment:
Is this actually needed? I thought `restore-keys` are additional options and only used if there's no hit for `key`. 
Has this changed over time?

Reply:
You're totally right, it does try the key first - I missed that, I thought it's just using `restore-keys` and since we were only using a generic version I've added a more precise one. But, reading the [caching docs](https://docs.github.com/en/actions/reference/workflows-and-actions/dependency-caching) again shows that it does use the `key` first. I'll remove these extra lines - thanks!